x-pack/plugins/cases/server/routes/api/__fixtures__/mock_saved_objects.ts

```diff
@@ -29,7 +28,7 @@ export const mockCases: Array<SavedObject<ESCaseAttributes>> = [
         id: 'none',
         name: 'none',
         type: ConnectorTypes.none,
-        fields: [],
+        fields: null,
```

Comment:
`fields: []` is the Elasticsearch representation when it is empty. Once it leaves the service layer it is transformed to `null`.